apps/server/src/routes/chat.ts

```
@@ -1,112 +1,471 @@
-      schema: z.object({
-        query: z.string(),
-      }),
+export class ZeroAgent extends AIChatAgent<typeof env> {
+  auth: SimpleAuth;
+  driver: MailManager | null = null;
+  constructor(ctx: DurableObjectState, env: Env) {
+    super(ctx, env);
+    this.auth = createSimpleAuth();
+  }
+
+  private getDataStreamResponse(onFinish: StreamTextOnFinishCallback<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)